ToDo.DataAccess/ToDo.DateAccessCore/DataBase/ToDoDbContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ToDo.DataAccess.Models;
+
+namespace ToDo.DataAccess.DataBase
```

Comment:
I asked to change project. Not to rename, not to add new one. This project should be named ToDo.DataAccess.

Reply:
Re-created.